assets/js/frontend/checkout.js

```diff
@@ -55,17 +55,21 @@ jQuery( function( $ ) {
 				$( 'input#createaccount' ).change( this.toggle_create_account ).change();
 			}
 		},
-		init_payment_methods: function() {
+		init_payment_methods: function( selectedPaymentMethod ) {
 			var $payment_methods = $( '.woocommerce-checkout' ).find( 'input[name="payment_method"]' );
 
-			// If there is one method, we can hide the radio input
-			if ( 1 === $payment_methods.length ) {
-				$payment_methods.eq(0).hide();
-			}
+			if ( selectedPaymentMethod ) {
```

Comment:
What if there is only 1 method, but that method is 'selected'? Does this still handle that? I ask because the radio is still there, just hidden if there is one method only.